core/src/cluster_slots.rs

```diff
@@ -192,9 +194,15 @@ impl ClusterSlots {
         let slot_peers = slot_peers.read().unwrap();
         repair_peers
             .iter()
-            .map(|peer| slot_peers.get(&peer.id).cloned().unwrap_or(0))
+            .map(|peer| slot_peers.get(&peer.id).is_some())
```

Comment:
`.contains_key(..)` instead of `.get(...).is_some()`:
https://doc.rust-lang.org/std/collections/hash_map/struct.HashMap.html#method.contains_key